app/components/publications/status_totals_component.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="govuk-grid-column-one-half">
```

Comment:
I think we need to add a class name for this specific component. `status-totals-component` or whatever.